_source/_posts/2022-05-04-angular-microfrontend-deploy.md

```diff
@@ -0,0 +1,286 @@
+
+This post is part two in a series where we'll build an e-commerce site using Angular using micro-frontends. We're using Webpack 5 with Module Federation to support wiring the micro-frontends together, demonstrate sharing authenticated state between the different front-ends, prepare for deployment using dynamic module loading, and deploy it all to Vercel's free hosting plan. 
+
+In this second post in the series, we're building on the site we created in the first post. Check out [How to Build Micro-Frontends Using Module Federation in Angular](/blog/2022/05/02/angular-microfrontend-auth). Let's continue working on our cupcake e-commerce site by adding dynamic module loading, securing unprotected routes, and setting up deploying the site in Vercel. 🎉
+
+In the end, you'll have an app that looks like this publicly available through Vercel:
+
+{% img blog/angular-microfrontend-deploy/final-app.gif alt:"Animated gif showing the final project with micro-frontends that handle the cupcake shopping basket, authentication, and profile information" width:"800" %}{: .center-image }
+
+**Prerequisites**
+
+- [Node](https://nodejs.org/) This project was developed using Node v16.14 with npm v8.5 
+- [Angular CLI](https://angular.io/cli)
+- [GitHub Account](https://github.com/)
+- [Vercel Account](https://vercel.com/) connected through your GitHub credentials for automating deployment.
+
+{% include toc.md %}
+
+## Review the Angular micro-frontends project using Webpack and Module Federation
+
+Let's start by refreshing our memories about where we left off—dust off your project from the first post. Just like last time, we'll need both IDE and the terminal. 
+
+We have the host application, `shell`, and two micro-frontend remotes, `mfe-basket` and `mfe-profile`. We're using `@angular-architects/module-federation` to help facilitate the Module Federation plugin configuration. The cupcake basket functionality code resides in the shared library, and we share authenticated state using Okta's SDKs. 
+
+Serve all of the Angular applications in the project by running:
+
+```shell
+npm run run:all
+```
+
+When you run the project, you should see all the beautiful cupcakes this store sells, be able to sign in, view your profile, add items to your basket, view your cupcake basket, and sign out. All the basics for handling your sweet treat needs!  
+
+> **Note**: _If you only want to follow the tutorial in this post and skip the first, you can clone the code sample repo to get going. You will be missing quite a bit of context, though. Use the following commands in this case:_
+```shell
+git clone https://github.com/oktadev/okta-angular-microfrontend-example.git
+cd okta-angular-microfrontend-example
+git checkout local
```

Comment:
You can simplify the clone command to use the branch if you want to combine steps:

```
git clone -b local https://github.com/oktadev/okta-angular-microfrontend-example.git

```